### PR TITLE
forgot a module.exports

### PR DESCRIPTION
**Pull request by tuna2134**

### CODE REVIEW

File: src/.vuepress/locales/details/locale_detail_ja.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const sidebar_ja = require("./sidebar/sidebar_ja");
 const nav_ja = require("./nav/nav_ja");
 
-let ja_locales = {
+module.exports = ja_locales = {
```
Comment (ahopkins) [inline src/.vuepress/locales/details/locale_detail_ja.js:4]: This is not valid JS. I think you meant to remove `ja_locales = `

**Review by ahopkins — CHANGES_REQUESTED** [review]

**Comment by tuna2134** [comment]
Fixed

File: src/.vuepress/locales/details/locale_detail_ja.js (hunk 1 above)
Comment (tuna2134) [reply src/.vuepress/locales/details/locale_detail_ja.js]: Check please

**Comment by tuna2134** [comment]
@ahopkins check please

**Review by ahopkins — APPROVED** [review]

**Pull request merged**